src/main/scala/is/hail/expr/Relational.scala

```diff
@@ -188,6 +188,14 @@ case class MatrixValue(
       }
     }
   }
+
+  def colsRVD(): RVD = {
+    val hc = HailContext.get
+    val signature = typ.colType
+
+    new UnpartitionedRVD(signature, hc.sc.parallelize(colValues.value.map(_.asInstanceOf[Row]))
```

Comment:
Convert to an array first.

@catoverdrive is this possibly dangerous if there's an unsafe row hiding in there?

Reply:
There can't be. I made colValues safe, I'm pretty sure